examples/common/python/connectors/tcf_connector.toml

```diff
@@ -33,12 +33,12 @@ direct_registry_contract_file = "examples/common/python/connectors/ethereum/cont
 worker_registry_contract_file = "examples/common/python/connectors/ethereum/contracts/WorkerRegistry.sol"
 #Deployed contract address of direct registry contract address.
 #Initial deploy the contract using eth_cli.py to get the this address
-direct_registry_contract_address = "0x9Be28B132aeE1b2c5A1C50529a636cEd807842cd"
+direct_registry_contract_address = ""
```

Comment:
If we set the direct_registry_contract_address to empty our unit tests won't run. That's why we added common shared account for testing contracts.

Reply:
That makes sense and addresses of runnable pre-installed contracts can be reserved.

Reply:
It shouldn't fail we are using those contract instances for our testing. It is shared account, if you want test with specific account then update with your credentials and contract instances locally.

Reply:
According to the given ethereum contracts, most functions must be called by the contract deployer. If the these default address are reserved, other users must deploy their own contracts and replace the related addresses in the config file.